python/whylogs/experimental/constraints_generation/condition_counts.py

```diff
@@ -0,0 +1,24 @@
+from typing import List
+
+from whylogs.core.constraints.factories import condition_meets
+from whylogs.core.constraints.metric_constraints import MetricConstraint
+from whylogs.core.view.column_profile_view import ColumnProfileView
+
+
+def generate_column_condition_count_constraints(
+    column_name: str, column_profile: ColumnProfileView
+) -> List[MetricConstraint]:
+    """Generates constraints for the condition count metrics of a column.
+    Parameters
+    ----------
+    column_name : str
+        Name of the column
+    column_profile : ColumnProfileView
+        Profile of the column
+    """
+    constraints = []
+    condition_count_metric = column_profile.get_metric("condition_count")
+    for condition, matches in condition_count_metric.matches.items():
+        if matches.value == condition_count_metric.total.value:
```

Comment:
I would consider the case where matches.value == 0 is also a strong signal that this is condition count could be considered a constraint. e.g. "No emails" kind of thing?

Reply:
That's a good suggestion!
Added it as an issue to do in a follow-up:
https://github.com/whylabs/whylogs/issues/1164